website/src/pages/Blog/Tag/index.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from "react";
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import {
+  Container,
+  Grid
+} from "@material-ui/core";
+import Footer from "../../../components/Footer";
+import { METADATA_TYPES } from "../../../constants";
+import Sponsor from "../../../components/Sponsor";
+import Pagination from "@material-ui/lab/Pagination";
+import { pageCount } from "../../../utils/getPageCount";
+import SeoJson from "../../../resources/seo.json";
+import { useCurrentHost } from "../../../hooks/useCurrentHost";
+import { Metadata } from "../../../components/Metadata";
+import BlogCard from "../../../components/BlogCard";
+import { useTag } from "../../../hooks/extractBlogPath";
+import { useHistory } from "react-router-dom";
+
+interface blog { 
+  title: string;
+  author: string;
+  excerpt: string;
+  author_info: string;
+  date: string;
+  tags: Array<string>;
+  content: string;
+  id: number;
+  slug: string; 
+}
+
+const Blog: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const { currentOrigin } = useCurrentHost();
+  const [filteredData, setFilteredData] = useState<blog[]>([
+    { title: "",
+      author: "",
+      excerpt: "",
+      author_info: "",
+      date: "",
+      tags: [],
+      content: "",
+      id: 0,
+      slug: "" }
+  ]);
+  const selectedTag = useTag();
+  const itemsPerPage = 6;
+  const [page, setPage] = React.useState<number>(1);
+  const history = useHistory();
+
+  const fetchBlogs = async () => {
+    const { default: blogs } = await import(`../../../posts.json`);
+    const filteredData = blogs.filter(
+      (blog: blog) => blog.tags.find((tag: string) => tag.toLowerCase() === selectedTag?.toLowerCase())
```

Comment:
blog: blog ==> please change the interface name

Reply:
blog: BlogItem